.github/CHANGELOG.md

```diff
@@ -465,6 +465,9 @@
 
 <h3>Improvements</h3>
 
+* The device test suite has been expanded to cover more qubit operations and observables.
+  [(#1510)](https://github.com/PennyLaneAI/pennylane/pull/1510)
+
```

Comment:
This entry will now belong to the v0.18.0 part of the changelog (top of the file). Don't forget to add yourself as a contributor too. :)

Reply:
Good spot!